example/generate_randoms.py

```diff
@@ -29,6 +29,21 @@ def write_i64(arr, name):
   pq.write_table(table, f'data/parquet/i64_{name}.parquet', compression='NONE')
   pq.write_table(table, f'data/snappy_parquet/i64_{name}.snappy.parquet', compression='snappy')
 
+def write_bool8(arr, name):
+  if arr.dtype != np.int8:
+    floored = np.floor(arr).astype(np.int8)
+  else:
+    floored = arr
+  ints = [str(x) for x in floored]
+  joined = '\n'.join(ints)
+  with open(f'data/txt/bool8_{name}.txt', 'w') as f:
+    f.write(joined)
+  with open(f'data/binary/bool8_{name}.bin', 'wb') as f:
```

Comment:
How about we instead pack the booleans into bytes? The raw data should in principle be 1/8 as big. But this is just for the example, so not totally essential

Reply:
Correct me if I'm wrong, but I was thinking the booleans are already packed into 8-bit bytes because of this line `floored = np.floor(arr).astype(np.int8)`

Reply:
Oh what I mean is that we can stuff 8 booleans into a single byte when storing on disk in a binary format. Not necessary to do this, and I can see an argument for leaving it as is.